runtime/src/main/resources/osc-huaweicloud.json

```diff
@@ -1,14 +1,15 @@
 {
-  "name": "open-services-cloud",
+  "name": "osc-hc",
```

Comment:
how about using `osc-huaweicloud` instead of `osc-hc`, to make it more recognizable.

Reply:
Good idea. I wanted something "short", but I agree. Let me update to use full qualified name ;)

Reply:
not sure if `hc` can be a short name for huaweicloud :P